projects/wave-core-new/src/lib/operators/datatype.model.ts

```diff
@@ -254,29 +255,50 @@ export abstract class VectorDataType {
      * @return The name of the data type.
      */
     abstract getCode(): string;
+
+    /**
+     * @return the corresponding result type
+     */
```

Comment:
I guess the return type of the function reflects exactly the doc-comment, so you could remove it?

Reply:
94609d6c441271ef765979abea31b424f563ba74